modules/stochastic_tools/include/state_sim/StateProcessor.h

```diff
@@ -0,0 +1,39 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#ifndef STATEPROCESSOR_H
+#define STATEPROCESSOR_H
+
+#include <string>
+#include <vector>
+
+using namespace std;
```

Comment:
You don't want to import the whole standard namespace in most complex codes (MOOSE included). Instead if you need a method out of this namespace just refer to it by its long name (i.e.` std::foo()`)